### PR TITLE
feat(Catalog Management): regen after recent api changes

### PR DESCRIPTION
**Pull request by benbuchanan**
## PR summary

regenerating after recent api changes

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## Current vs new behavior  

adding new endpoints, removing some deprecated fields

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by benbuchanan** [comment]
```
integration
Ran 735 of 791 Specs in 45.839 seconds
SUCCESS! -- 735 Passed | 0 Failed | 0 Pending | 56 Skipped
PASS
ok  	github.com/IBM/platform-services-go-sdk/catalogmanagementv1	46.832s

example
Ran 723 of 753 Specs in 45.915 seconds
SUCCESS! -- 723 Passed | 0 Failed | 0 Pending | 30 Skipped
PASS
ok  	github.com/IBM/platform-services-go-sdk/catalogmanagementv1	46.958s
```

File: catalogmanagementv1/catalog_management_v1_examples_test.go [hunk 1]
```diff
@@ -309,6 +309,29 @@ var _ = Describe(`CatalogManagementV1 Examples Tests`, func() {
 			Expect(offering).ToNot(BeNil())
 		})
 
+		It(`GetOfferingStats request example`, func() {
+			fmt.Println("\nGetOfferingStats() result:")
+			// begin-get_offering
```
Comment (padamstx) [inline catalogmanagementv1/catalog_management_v1_examples_test.go:314]: this should be 
```suggestion
			// begin-get_offering_stats
```

**Review by padamstx — CHANGES_REQUESTED** [review]
I approved too quickly before :)
The examples code needs a couple of small changes.

File: catalogmanagementv1/catalog_management_v1_examples_test.go [hunk 2]
```diff
@@ -325,7 +325,7 @@ var _ = Describe(`CatalogManagementV1 Examples Tests`, func() {
 			b, _ := json.MarshalIndent(offering, "", "  ")
 			fmt.Println(string(b))
 
-			// end-get_offering
+			// end-get_offering-stats
```
Comment (padamstx) [inline catalogmanagementv1/catalog_management_v1_examples_test.go:328]:
```suggestion
			// end-get_offering_stats
```

**Review by padamstx — CHANGES_REQUESTED** [review]

**Review by padamstx — APPROVED** [review]
LGTM

**Pull request merged**